modules/twinklexfd.js

```diff
@@ -533,6 +574,12 @@ Twinkle.xfd.callbacks = {
 				params.xfdcat + (params.reason ? ': ' + Morebits.string.formatReasonText(params.reason) : '.') + ' ~~~~';
 			// U+00A0 NO-BREAK SPACE; U+2013 EN RULE
 		}
+		if (venue === 'rm') {
+			// even if invoked from talk page, propose the subject page for move
+			var pageName = new mw.Title(Morebits.pageNameNorm).getSubjectPage().toText();
+			return '{{subst:' + (params.rmtr ? 'RMassist|1=' : 'Requested move|current1=') + pageName + '|' + params.newname + '|reason=' + params.reason + '}}';
```

Comment:
`{{RMassist}}` craps out when the second parameter isn't provided explicitly but the first is, e.g. `{{subst:RMassist|1=Macbeth|Earth}}`.  I *think* that's something that should probably be handled on the template side of things, but if it isn't, `RMassist` either needs both specified (`1=`, `2=`) or neither.

Reply:
fixed. There are some pages with = in name, so we can't use unnamed and unnumbered parameters.

Reply:
Ah yeah, that'd be it.  [Name all the parameters!](https://github.com/azatoth/twinkle/pull/546) is a fair rallying cry.